### ISSUE TITLE
Skeleton requires Plack/Middleware/Deflater.pm but Dancer2 does not install it

### ISSUE THREAD

**Issue by szabgab** [PerlDancer/Dancer2#1381]
On a clean machine I've just installed Dancer2 v 0.205001 created a new application using `dancer2 gen -a App::Demo`, changed directory `cd App-Demo` and then tried to run the application: `plackup bin/app.psgi` as recommended by the `dancer2` command.

I got:

```
Error while loading /home/ubuntu/App-Demo/bin/app.psgi: Can't locate Plack/Middleware/Deflater.pm in @INC (you may need to install the Plack::Middleware::Deflater module) (@INC contains: /home/ubuntu/App-Demo/bin/../lib /home/ubuntu/perl-5261/lib/site_perl/5.26.1/x86_64-linux /home/ubuntu/perl-5261/lib/site_perl/5.26.1 /home/ubuntu/perl-5261/lib/5.26.1/x86_64-linux /home/ubuntu/perl-5261/lib/5.26.1) at /home/ubuntu/perl-5261/lib/site_perl/5.26.1/Plack/Util.pm line 26.
```

I think either the skeleton should not require `Plack::Middleware::Deflater` or Dancer2 should include it as a dependency.

**Comment by nunorc** [comment]
It seems to me that that snippet that enables the middleware is included in that section of the skeleton by mistake. The basic app should not enable the middleware, since the module is not a Dancer2 dependency.

PR https://github.com/PerlDancer/Dancer2/pull/1382 fixes this issue, someone else should double check this of course.

**Comment by ambs** [comment]
Fixed by #1357.
